src/components/shared/HeaderTitle.tsx

```diff
@@ -0,0 +1,19 @@
+import {
+  Text,
+  StyleSheet,
+} from 'react-native';
+
+import GlobalStyles from '../../GlobalStyles';
+
+export default (props) => {
```

Comment:
Another comment for code standardization to be documented/adhered to.
Using `export default (props) =>` as opposed to `const ComponentName = (props) => ... export default ComponentName`

Reply:
Addressed in #13 (another low-Pri)